optimizely/event_builder.py

```diff
@@ -172,19 +182,39 @@ def _get_attributes(self, attributes):
     if not attributes:
       return []
 
-    for attribute_key in attributes.keys():
+    for attribute_key in sorted(attributes.keys()):
```

Comment:
why do we need to sort?

Reply:
attributes is a dict. Order of keys is not guaranteed to be same for all python versions. Hence for eventpayload unit tests to pass for all versions, we need to ensure that they get appended in the same order.

Reply:
Why does ordering matter? Let us not do this. It seems unnecessary.

Reply:
@aliabbasrizvi  attributes is a dict. Order of keys is not guaranteed to be same for all python versions. Hence for eventpayload unit tests to pass for all versions, we need to ensure that they get appended in the same order.
https://travis-ci.org/optimizely/python-sdk/builds/389076889